specification/vmware/resource-manager/Microsoft.AVS/stable/2023-03-01/vmware.json

```diff
@@ -5902,12 +5902,46 @@
         "managementCluster",
         "networkBlock"
       ],
-      "allOf": [
-        {
-          "$ref": "#/definitions/PrivateCloudUpdateProperties"
```

Comment:
The solution is remove this `allOf`, then copy all of the properties in, in the correct order. `extendedNetworkBlocks` is moved to the end.